qiskit/circuit/quantumcircuit.py

```diff
@@ -23,6 +23,7 @@
 from collections import OrderedDict, defaultdict
 from typing import Union
 import numpy as np
+import qiskit as qk
```

Comment:
Why do we need this import?

Reply:
I can't remember why I added it :D It's now removed.